src/Bonnier/WP/WypeResetpassword/Http/Routes/SubscriberReducedPricePage.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Bonnier\WP\WypeResetpassword\Http\Routes;
+
+use Bonnier\WP\WypeResetpassword\Plugin;
+use Bonnier\WP\WypeResetpassword\Services\BmdEmailFromSubscriptionService;
+use Bonnier\WP\WypeResetpassword\Services\BmdValidateLoginService;
+
+class SubscriberReducedPricePage extends BasePageRoute
+{
+    const PAGE_URI = '/subscriber-reduced-price';
+    const PAGE_FILE = 'subscriber-reduced-price.php';
+
+    public static $inputError = false;
+
+    public static function isAuthenticated() {
+        if(isset($_POST['email_or_subscription_number'], $_POST['postal_code'])
+            && !empty($_POST['email_or_subscription_number'])
+            && !empty($_POST['postal_code']))
+        {
+            $postal_code = $_POST['postal_code'];
```

Comment:
Hold dig gerne til camelCase for variabler, da vi prøver at følge PS-2 code style :)